lib/runner/VirtualDeviceInvoker.js

```diff
@@ -88,15 +92,18 @@ module.exports = class VirtualDeviceInvoker extends Invoker {
     }
 
     convertUtterance(interaction) {
+        const invocationName = interaction.test.testSuite.invocationName;
         let utterance = interaction.utterance;
         if (interaction.utterance === "LaunchRequest") {
-            utterance = `open ${interaction.test.testSuite.invocationName}`;
+            utterance = `open ${invocationName}`;
         } else if (interaction.utterance.startsWith("AudioPlayer.")) {
             utterance = undefined;
         } else if (interaction.utterance === "SessionEndedRequest") {
             utterance = "exit";
         } else {
-            if (interaction.relativeIndex === 0) {
+            const launchRequestRegex = new RegExp(`^(ask|open|launch|talk to|tell) ${invocationName}`, "i");
+            const isLaunchRequest = launchRequestRegex.test(interaction.utterance);
```

Comment:
I would call this includesInvocationName - just because saying "tell <invocationName> to <intent>" does include the invocationName but it is not a launchrequest (it will actually directly invoke the named intent)